packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -104,6 +104,16 @@ function createMockRedis() {
 
 mockConfig.redis = mockRedisConfig;
 
+/**
+ * To prevent the modification of the test objects loaded, which can impact other tests referencing the objed,
+ * a deep copy of the object can be created which uses the test object as a template
+ *
+ * @param {Object} object
+ */
+function deepCopy(object) {
```

Comment:
nit: `lodash` has a `cloneDeep`, and we use some lodash packages elsewhere for testing. Might be useful to bring that in here. This also seems fine.